packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -62,6 +62,8 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
           id: true,
           slug: true,
           metadata: true,
+          name: true,
+          logo: true,
```

Comment:
is it fine to fetch logo here? https://github.com/calcom/cal.com/issues/9669

Reply:
Nice catch !! @Udit-takkar 
We should support an org avatar endpoint and then we don't need to return the logo from here,

Reply:
https://app.gitbook.com/o/6snd8PyPYMhg0wUw6CeQ/s/VXRprBTuMlihk37NQgUU/~/changes/101/engineering/performance-best-practices#say-no-to-big-payloads-in-endpoints
Added in the Performance best practices Change Request on gitbook

We should probably have a way to detect and prevent that. Maybe we can have a middleware that runs on dev and that can detect data URI in a property and throw error or warning. 
Making it dev only would ensure no performance impact in production due to middleware.